src/interactions.rs

```diff
@@ -1,31 +1,50 @@
 use crate::github::{GithubClient, Issue};
 use std::fmt::Write;
 
-pub struct ErrorComment<'a> {
+pub struct Comment<'a> {
     issue: &'a Issue,
     message: String,
 }
 
+impl<'a> Comment<'a> {
+    pub fn new<T>(issue: &'a Issue, message: T) -> Comment<'a>
+    where
+        T: Into<String>,
+    {
+        Comment {
+            issue,
+            message: message.into(),
+        }
+    }
+
+    pub async fn post(&self, client: &GithubClient) -> anyhow::Result<()> {
+        self.issue.post_comment(client, &self.message).await
+    }
+}
```

Comment:
There's no need for a separate Comment struct, we can directly call post_comment. ErrorComment exists because it does more than that.

Reply:
It turns out that `Comment` struct wasn't even being used.... I must have added it for something that I didn't up needing.